### PR TITLE
Refactoring/#1094/settings

### PR DESCRIPTION
**Pull request by yi-cro**
Issue: https://github.com/crypto-com/chain-desktop-wallet/issues/1094

Task(s)
 - Move storage directory into service directory as it contains storage service thus helps with specifics
 - Adjust other files according to the change of the storage directory

### CODE REVIEW

**Review by crypto-matto — COMMENTED** [review]
## Suggestion
- It is not appropriate to put `getAssetConfigFromWalletConfig()` as a `component`. 
You may either 
  - create a new `utils` folder and put `getAssetConfigFromWalletConfig()` inside a `utils.tsx`
  - or you just put `getAssetConfigFromWalletConfig()` inside `FormSettings.tsx` as I'm seeing it's only called in this file.

**Comment by yi-cro** [comment]
@crypto-matto 
`getAssetConfigFromWalletConfig()` is being called in two files from what I remember thus I separated it into a single file that can be called

I could just create a utils.tsx if you prefer
I will have to that in the next push later as currently I am working on other parts of project.

File: src/pages/staking/components/FormDelegationOperations.tsx [hunk 1]
```diff
@@ -154,22 +154,22 @@ export const FormDelegationOperations = props => {
       let broadcastedTransaction: BroadCastResult | null = null;
 
       if (delegationActionType === StakingActionType.UNDELEGATE) {
-        const undelegateAmount = form.getFieldValue('undelegateAmount');
+        // const undelegateAmount = form.getFieldValue('undelegateAmount');
```
Comment (crypto-matto) [inline src/pages/staking/components/FormDelegationOperations.tsx:157]: Remove the unused code

File: src/pages/staking/components/FormDelegationOperations.tsx [hunk 2]
```diff
@@ -154,22 +154,22 @@ export const FormDelegationOperations = props => {
...
-          amount: undelegateAmount,
+          amount: undelegateFormValues.undelegateAmount,
           asset: userAsset,
           memo: '',
           decryptedPhrase,
           walletType,
         });
       } else if (delegationActionType === StakingActionType.REDELEGATE) {
-        const redelegateAmount = form.getFieldValue('redelegateAmount');
+        // const redelegateAmount = form.getFieldValue('redelegateAmount');
```
Comment (crypto-matto) [inline src/pages/staking/components/FormDelegationOperations.tsx:167]: Remove the unused code

Comment (yi-cro) [reply src/pages/staking/components/FormDelegationOperations.tsx]: Removed

File: src/pages/staking/components/FormDelegationOperations.tsx (hunk 1 above)
Comment (yi-cro) [reply src/pages/staking/components/FormDelegationOperations.tsx]: Removed

**Review by crypto-matto — APPROVED** [review]

**Pull request merged**